<?xml version="1.0" encoding="UTF-8"?>
<testng-results skipped="0" failed="1" ignored="0" total="3" passed="2">
  <reporter-output>
  </reporter-output>
  <suite name="Suite" duration-ms="33504" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:50 IST">
    <groups>
    </groups>
    <test name="Test2" duration-ms="32233" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:49 IST">
      <class name="testcases.HomePageTestCase01">
        <test-method status="PASS" signature="start()[pri:0, instance:testcases.HomePageTestCase01@710636b0]" name="start" is-config="true" duration-ms="21179" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:38 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method status="PASS" signature="basePageNavigation(java.lang.String,java.lang.String)[pri:0, instance:testcases.HomePageTestCase01@710636b0]" name="basePageNavigation" duration-ms="7672" started-at="2022-03-04T12:20:38 IST" data-provider="getData" finished-at="2022-03-04T12:20:45 IST">
          <params>
            <param index="0">
              <value>
                <![CDATA[abcd]]>
              </value>
            </param>
            <param index="1">
              <value>
                <![CDATA[1234]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.HomePageTestCase01@710636b0]" name="tearDown" is-config="true" duration-ms="3318" started-at="2022-03-04T12:20:45 IST" finished-at="2022-03-04T12:20:49 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.HomePageTestCase01 -->
    </test> <!-- Test2 -->
    <test name="Test1" duration-ms="33504" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:50 IST">
      <class name="testcases.RegisterTestCase01">
        <test-method status="PASS" signature="start()[pri:0, instance:testcases.RegisterTestCase01@1033576a]" name="start" is-config="true" duration-ms="20168" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:37 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method status="PASS" signature="basePageNavigation()[pri:0, instance:testcases.RegisterTestCase01@1033576a]" name="basePageNavigation" duration-ms="5237" started-at="2022-03-04T12:20:37 IST" finished-at="2022-03-04T12:20:42 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- basePageNavigation -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.RegisterTestCase01@1033576a]" name="tearDown" is-config="true" duration-ms="7600" started-at="2022-03-04T12:20:42 IST" finished-at="2022-03-04T12:20:50 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.RegisterTestCase01 -->
    </test> <!-- Test1 -->
    <test name="Test3" duration-ms="33362" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:50 IST">
      <class name="testcases.ValidateTitleCase">
        <test-method status="PASS" signature="start()[pri:0, instance:testcases.ValidateTitleCase@7bd7d6d6]" name="start" is-config="true" duration-ms="20133" started-at="2022-03-04T12:20:16 IST" finished-at="2022-03-04T12:20:36 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- start -->
        <test-method status="FAIL" signature="validateTitle()[pri:0, instance:testcases.ValidateTitleCase@7bd7d6d6]" name="validateTitle" duration-ms="1022" started-at="2022-03-04T12:20:37 IST" finished-at="2022-03-04T12:20:38 IST">
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[expected [ ] but found [Login | Salesforce]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: expected [ ] but found [Login | Salesforce]
at org.testng.Assert.fail(Assert.java:99)
at org.testng.Assert.failNotEquals(Assert.java:1037)
at org.testng.Assert.assertEqualsImpl(Assert.java:140)
at org.testng.Assert.assertEquals(Assert.java:122)
at org.testng.Assert.assertEquals(Assert.java:629)
at org.testng.Assert.assertEquals(Assert.java:639)
at testcases.ValidateTitleCase.validateTitle(ValidateTitleCase.java:37)
at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.lang.reflect.Method.invoke(Method.java:498)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)
at org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)
at org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)
at org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)
at org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)
at org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)
at java.util.ArrayList.forEach(ArrayList.java:1259)
at org.testng.TestRunner.privateRun(TestRunner.java:794)
at org.testng.TestRunner.run(TestRunner.java:596)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:377)
at org.testng.SuiteRunner.access$000(SuiteRunner.java:28)
at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:418)
at org.testng.internal.thread.ThreadUtil.lambda$execute$0(ThreadUtil.java:64)
at java.util.concurrent.FutureTask.run(FutureTask.java:266)
at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
at java.lang.Thread.run(Thread.java:748)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- validateTitle -->
        <test-method status="PASS" signature="tearDown()[pri:0, instance:testcases.ValidateTitleCase@7bd7d6d6]" name="tearDown" is-config="true" duration-ms="8788" started-at="2022-03-04T12:20:41 IST" finished-at="2022-03-04T12:20:50 IST">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- tearDown -->
      </class> <!-- testcases.ValidateTitleCase -->
    </test> <!-- Test3 -->
  </suite> <!-- Suite -->
</testng-results>
